src/Cake.Common/Tools/XUnit/XUnitRunner.cs

```diff
@@ -119,5 +122,12 @@ protected override System.Collections.Generic.IEnumerable<string> GetToolExecuta
         {
             return new[] { "xunit.console.clr4.exe" };
         }
+
+        private static FilePath GetReportName(IReadOnlyList<FilePath> assemblies)
```

Comment:
Can we avoid duplication of this method in both the `XUnitRunner` and `XUnit2Runner`?